fossevents/events/decorators.py

```diff
@@ -0,0 +1,20 @@
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+
+from . import models
+
+
+def match_token(function):
+    """
+    Decorator for views that checks that the user is logged in, redirecting
+    to the log-in page if necessary.
```

Comment:
There is no login in the app, it's just checks the authorization for a particular page, need to update the comment here. No redirection to login or anywhere please. If the token, is present and valid the edit view open that's it. You can just have a url for the form:`/edit/event/<token>`.